### DISCUSSION TITLE
Icon with info

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#15998] [Archive]
Hi Team,

Is there any option to display icon with info in each field lable in collection.

Thanks,
Sandeep

**Comment by rijkvanzanten** [comment]
Nope. Field labels are shown underneath the field

**Reply by sandyksit** [reply]
Any possibility in future?

**Reply by rijkvanzanten** [reply]
Maybe! If there's enough upvotes on a feature request for such a thing where it's clear that this is useful to have for 80%+ of the community, we can add it to core 🙂
